ethcore/vm/src/schedule.rs

```diff
@@ -113,6 +113,8 @@ pub struct Schedule {
 	pub kill_dust: CleanDustMode,
 	/// Enable EIP-86 rules
 	pub eip86: bool,
+	/// Is wasm active
+	pub wasm_activated: bool,
```

Comment:
Maybe `wasm: Option<WasmCosts>` will be more future proof?

Reply:
how do you imagine this line then:

```
		if self.wasm || block_number >= self.wasm_activation {
			schedule.wasm_activated = true;
		}

```

in `CoimmonParams::update_schedule`

`schedule.wasm = Some(...<WHAT?>)` ?

where the prices will be held until they are suddenly active?

Reply:
On the second thought, no need to keep them, they can be just default :)

Reply:
Yeah, they can be default for now, but later probably come from spec (that's why I suggested merging them with the flag).